src/main/java/qna/domain/Question.java

```diff
@@ -72,22 +74,28 @@ public void setDeleted(boolean deleted) {
     }
 
     public List<Answer> getAnswers() {
-        return answers;
+        return answersRelatedQuestion.getValue();
     }
 
     public void addAnswer(Answer answer) {
-        this.answers.add(answer);
+        this.answersRelatedQuestion.add(answer);
         answer.setQuestion(this);
     }
 
-    @Override
-    public String toString() {
-        return "Question{" +
-                "id=" + id +
-                ", title='" + title + '\'' +
-                ", contents='" + contents + '\'' +
-                ", writer=" + writer +
-                ", deleted=" + deleted +
-                '}';
+    public List<DeleteHistory> deleteAndCreateDeleteHistory(User loginUser) {
```

Comment:
개인적으로 퍼블릭 메서드 명을 정할때는 협력에 맞게 추상적인 작성하는 것을 선호하는데요.그 이유는 내부 구현이 바뀌더라도, 메서드 명을 유지할 수 있기 때문입니다.메서드 명을 유지한다는 것은 생각보다 단순해보이기도하지만, 처음 정의한 협력을 바꾸지 않는다는 관점에서 바라보면 굉장히 중요하게 느껴지기도 하네요 🤔

조금 극단적이게 생각해보면 질문을 삭제할때 답변을 삭제하지 않도록 요구사항이 변경될 수도 있을 것 같아요 😄

> 구현과 관련된 모든 정보를 캡슐화하고 객체의 퍼블릭 인터페이스에는 협력과 관련된 의도만을 표현해야 한다.

```suggestion
    public List<DeleteHistory> delete(User loginUser) {
```